scripts/git_pr/add_highlighting.js

```diff
@@ -0,0 +1,72 @@
+// ------------------------------------------------------------------
+// highlights selected word
+function addHighlighting() {
+  // checks what is selected
+  function getSelectedText() {
+    var text = '';
+    if (typeof window.getSelection != 'undefined') {
+      text = window.getSelection().toString();
+    } else if (typeof document.selection != 'undefined' && document.selection.type == 'Text') {
+      text = document.selection.createRange().text;
+    }
+    return text;
+  }
+
+  // propagates in DOM until it finds parent ancestor
+  function findAncestor(el, cls) {
+    while ((el = el.parentElement) && !el.classList.contains(cls));
+    return el;
+  }
+
+  // get all elements with desired text
+  function findElementsWithText(text, elements) {
+    return [].filter.call(elements, function(element) {
```

Comment:
`I would add Array.from(elements)` because as is, this will break on MS Edge (microsoft man strikes again!!!)

Reply:
This is for Chrome extension, only Chrome will see this beautiful thing! Microsoft man doesn't even have extensions!